src/test/FuzzerImpl.cpp

```diff
@@ -800,7 +805,11 @@ TransactionFuzzer::initialize()
 
         for (auto const& offer : offerParameters)
         {
-            auto op = txtest::manageOffer(0, offer.mBid, offer.mSell,
+            auto op =
+                offer.mPassive
+                    ? txtest::createPassiveOffer(offer.mSell, offer.mBid,
+                                                 offer.mPrice, offer.mAmount)
+                    : txtest::manageOffer(0, offer.mBid, offer.mSell,
```

Comment:
These lines can't both be right. The assets come in the same order in both functions.

Reply:
> These lines can't both be right. The assets come in the same order in both functions.

Good catch.  No idea how I managed to do that.  Fixed.